site/content/en/docs/getting-started-deploy.md

```diff
@@ -107,6 +107,15 @@ $ kubectl create clusterrolebinding cluster-admin-binding-"<CLUSTER_USER>" \
 
 There are [relevant docs on Kubernetes Authentication](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#authentication-strategies) that may help if neither of the above work.
 
+### Create the namespace
```

Comment:
This isn't accurate. Unfortunately, prow must be deployed in the default namespace.

It's a bug that we observed recently and needs to be fixed.

@cjwagner Did we file a bug about the component that was trying to fetch something from the default namespace?

Reply:
I don't recall that issue. I guess this means the starter.yaml files are broken since they specify `namespace: prow`.

Reply:
I'm also not aware of any issue like that . OpenShift Prow is definitely not deployed in default namespace and I don't think we ever saw any issue related to that.

Reply:
I was following the conversation to see if I missed anything but just to add that we deployed Prow in `prow` namespace and all is well.